src/Parser/DeathKnight/Unholy/Modules/Features/RpPoolingDA.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+
+class RpPoolingDA extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.DARK_ARBITER_TALENT.id);
+  }
+
+  totalRunicPowerPooled = 0;
+  totalDarkArbiterCasts = 0;
+  currentRP = 0;
+  maxRP = 0;
+  
+   on_toPlayer_energize(event){
+	   this.currentRP = event.classResources[0]['amount']/10 || 0;
+	   this.maxRP = event.classResources[0]['max']/10;
+   }
+   
+  on_byPlayer_cast(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.DARK_ARBITER_TALENT.id){
+		this.totalDarkArbiterCasts++;
+		this.totalRunicPowerPooled += this.currentRP;
+		}
+	}
+
+  suggestions(when) {
+    const averageRpPooled = (this.totalRunicPowerPooled/this.totalDarkArbiterCasts).toFixed(1);
+    when(averageRpPooled).isLessThan(this.maxRP-20)
```

Comment:
A totally nit picky comment: in the new results rework where we're introducing the Checklist it's recommended to write suggestion thresholds differently. It might benefit to start doing this already so that you don't have to rewrite as much later when you want to implement the checklist. Here's an example:

https://github.com/WoWAnalyzer/WoWAnalyzer/blob/46d812bdda19ad4eb06e7d372913835e926b6e1d/src/Parser/Paladin/Holy/Modules/Talents/AuraOfSacrifice.js#L35-L57